Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.hxx

```diff
@@ -341,7 +341,12 @@ DisplacementFieldTransform<TParametersValueType, VDimension>::SetDisplacementFie
   }
   this->SetFixedParametersFromDisplacementField();
 }
-
+template <typename TParametersValueType, unsigned int VDimension>
+void
+DisplacementFieldTransform<TParametersValueType, VDimension>::SetDisplacementField(VectorImage<TParametersValueType, VDimension> * field)
+{
+  return;
```

Comment:
Shouldn't this do something? I guess that delegating to the other method does not compile?